app/components/CommentInput/CommentInputAuthor.jsx

```diff
@@ -64,11 +65,13 @@ CommentInputAuthor.propTypes = {
   anonymousProfilePicture: PropTypes.string,
   profilePicture: PropTypes.string.isRequired,
   selectPostingAs: PropTypes.func.isRequired,
+  is_public: PropTypes.bool.isRequired,
```

Comment:
You can delete the isRequired
```suggestion
  is_public: PropTypes.bool,
```